test_config/test_config.go

```diff
@@ -28,29 +28,36 @@ import (
 	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
 )
 
-var TestConfig *viper.Viper
 var DBConfig config.Database
 var TestClient config.Client
 var ABIFilePath string
 var wipeTableQueries []string
 
 func init() {
-	setTestConfig()
+	SetTestConfig()
 	setABIPath()
 }
 
-func setTestConfig() {
-	TestConfig = viper.New()
-	TestConfig.SetConfigName("testing")
-	TestConfig.AddConfigPath("$GOPATH/src/github.com/makerdao/vdb-mcd-transformers/environments/")
-	err := TestConfig.ReadInConfig()
-	if err != nil {
-		log.Fatal(err)
+func SetTestConfig() {
+	viper.AddConfigPath("$GOPATH/src/github.com/makerdao/vdb-mcd-transformers/environments/")
+
+	viper.SetConfigName("testDatabase")
+	mergeConfigErr := viper.ReadInConfig()
+	if mergeConfigErr != nil {
+		log.Fatal(mergeConfigErr)
+	}
+
+	viper.SetConfigName("mcdTransformers")
+	readConfigErr := viper.MergeInConfig()
+	if readConfigErr != nil {
+		log.Fatal(readConfigErr)
 	}
-	ipc := TestConfig.GetString("client.ipcPath")
-	hn := TestConfig.GetString("database.hostname")
-	port := TestConfig.GetInt("database.port")
-	name := TestConfig.GetString("database.name")
+
+	ipc := viper.GetString("client.ipcPath")
```

Comment:
Not your responsibility in this PR but I'm wondering if we can either remove the `ipc` + `TestClient` or use the `ipc` derived here in the places (like integration test config) where we're depending on this to grab the DB config

Reply:
I need to think on this a bit, so will probably do it as part of a new PR.